sys/src/cmd/screenlock.c

```diff
@@ -241,21 +240,23 @@ lockscreen(void)
 	procrfork(blanker, nil, 4096, RFFDG);
 	fd = open(pic, OREAD);
 	if(fd > 0){
+		draw(screen, screen->r, display->black, nil, ZP);
```

Comment:
If the open fails, I still want a black screen, because the text is white.  As it was, it would be white on white which isn't very helpful.